### ISSUE TITLE
"Alexa" wake word is not filtered from history/summary for commands via app

### ISSUE THREAD

**Issue by padrino-x** [Apollon77/ioBroker.alexa2#613]
When you give commands via the mobile app, by telling your phone "alexa, ..." the word "alexa" will show up in "summary" along with the command.
The same done from/for any echo device will filter the wake word. =)

**Comment by Apollon77** [comment]
Yes, basically only the configured wakeword of the device is filtered. if this is not alexa and you use alexa via mobile app the it is npt filtered because not considered as "the wake word". what else the adapter shpuld do? Always filter alexa?

**Comment by padrino-x** [comment]
Ok, as written in the forum (just putting it here for completeness), the wake word for the app is always "alexa", there is no way to change/configure it.

**Comment by padrino-x** [comment]
Just a little bumper the "stale bot" forces me to, so that the issue isn't auto-closed...

*bump* 😅

**Comment by Apollon77** [comment]
Please check github version
